### PR TITLE
75-data-minimization-challenge

### PR DESCRIPTION
**Pull request by EmanueleMusumeci**

### CODE REVIEW

File: challenges/data_minimization_challenge.tex [hunk 1]
```diff
@@ -1,3 +1,38 @@
...
+where:
+\begin{itemize}
+    \item $M$ is the number of matches played by team $t$
+\end{itemize}
+
+
+The final objective of each team is to minimize the $A^t$ average.
+
+\subsection{Exclusion from challenge}
+Being an in-game challenge, all teams participating in the main competition will automatically take part in this challenge if they score at least one goal during the whole duration of the preliminary tournament phases.
```
Comment (PatrickGoettsch) [inline challenges/data_minimization_challenge.tex:35]:
```suggestion
Being an in-game challenge, all teams participating in the main competition will automatically take part in this challenge if they score at least one goal during the whole duration of the preliminary tournament phase.
```

**Review by PatrickGoettsch — CHANGES_REQUESTED** [review]

File: challenges/data_minimization_challenge.tex (hunk 1 above)
Comment (EmanueleMusumeci) [reply challenges/data_minimization_challenge.tex:35]: I agree, my mistake :)

File: challenges/data_minimization_challenge.tex [hunk 2]
```diff
@@ -1,3 +1,38 @@
...
+
+$$ A^\text{t} = \frac{1}{M} \left( \sum_{m}^M R^\text{m} \right) $$
+
+where:
+\begin{itemize}
+    \item $M$ is the number of matches played by team $t$
+\end{itemize}
+
+
+The final objective of each team is to minimize the $A^t$ average.
```
[the remark this answers is not in the document]
Comment (EmanueleMusumeci) [reply challenges/data_minimization_challenge.tex:32]: Thanks :)

File: challenges/data_minimization_challenge.tex [hunk 3]
```diff
@@ -1,3 +1,38 @@
...
+$$ R^\text{m} = \frac{\sum_{i}^n b_i}{\sum_{i}^n t_i} $$
+
+where:
+\begin{itemize}
+    \item $b_i$ is the number of bytes sent by robot $i$
+    \item $t_i$ is the uptime of robot $i$
+\end{itemize}
+The sum takes into account all robots that were at some point active during the match, therefore also correctly taking into account substitutions and penalizations.
+
+An average is computed for each match while the final score $A^\text{t}$ for team $t$ will simply be the global average over all matches disputed by that team:
```
[the remark this answers is not in the document]
Comment (EmanueleMusumeci) [reply challenges/data_minimization_challenge.tex:22]: With global average I just meant the average over all matches but I think your edit makes it more readable :)

File: challenges/data_minimization_challenge.tex [hunk 4]
```diff
@@ -1,3 +1,38 @@
 % !TeX root = ../SPL-Challenges.tex
 % !TeX spellcheck = en_US
-\section{Data Minimization Challenge} % TODO: I hope there is a better name for this challenge
+\section{Network Usage Challenge} % TODO: I hope there is a better name for this challenge
+This challenge will take place throughout regular games and the score will be based on the quantity of bytes exchanged by active robots during matches.
```
[the remark this answers is not in the document]
Comment (EmanueleMusumeci) [reply challenges/data_minimization_challenge.tex:4]: I agree with this point. I don't think a discussion about it is needed in the next TC meeting and we might implement it right away but if a discussion is needed I'm open to this possibility. I'm applying your suggestion.

File: challenges/data_minimization_challenge.tex [hunk 5]
```diff
@@ -1,3 +1,38 @@
...
-\section{Data Minimization Challenge} % TODO: I hope there is a better name for this challenge
+\section{Network Usage Challenge} % TODO: I hope there is a better name for this challenge
+This challenge will take place throughout regular games and the score will be based on the quantity of bytes exchanged by active robots during matches. 
+
+
+\subsection{Challenge Goal}
+The purpose of this challenge is to reward good network data usage strategies that minimize the number of bytes sent by each active robot. Both the quantity of exchanged bytes and the uptime of active robots will be taken into consideration when computing the score.
+
+\subsection{Challenge Scoring}
+Network data usage will be evaluated by keeping track of the ratio $R$ between the cumulative count of bytes sent by all active robots in the team and the total uptime of robots in the team for SPL match $m$:
```
[the remark this answers is not in the document]
Comment (EmanueleMusumeci) [reply challenges/data_minimization_challenge.tex:11]: Thank you, I've spent some words to define this, could you give a look at line 18?

File: challenges/data_minimization_challenge.tex [hunk 6]
```diff
@@ -1,3 +1,38 @@
...
+
+$$ R^\text{m} = \frac{\sum_{i}^n b_i}{\sum_{i}^n t_i} $$
+
+where:
+\begin{itemize}
+    \item $b_i$ is the number of bytes sent by robot $i$
+    \item $t_i$ is the uptime of robot $i$, defined as the time during which robot $i$ is playing and is not penalized.
```
Comment (PatrickGoettsch) [inline challenges/data_minimization_challenge.tex:18]: I would phrase it a little bit differently:
```suggestion
    \item $t_i$ is the uptime of robot $i$, i.e., the duration robot $i$ is participating in the match and is not penalized.
```

**Pull request merged**